### PR TITLE
Fixing remaining issues from IESG eval

### PR DESCRIPTION
**Pull request by henkbirkholz**
based on: https://mailarchive.ietf.org/arch/msg/sacm/7h7j2WMCMi0fccrhy_I8uqff6bA/

### CODE REVIEW

File: draft-ietf-sacm-coswid.md [hunk 1]
```diff
@@ -965,7 +963,7 @@ The following describes each child item of this group.
 
 - resource-collection: The resource-collection group described in {{model-resource-collection}}.
 
-- date (index 35): The date and time the information was collected pertaining to the evidence item.
+- date (index 35): The date and time the information was collected pertaining to the evidence item in Epoch-Based Date/Time format as specified in Section 3.4.2. in {{RFC8949}}.
```
Comment (david-waltermire) [inline draft-ietf-sacm-coswid.md]:
```suggestion
- date (index 35): The date and time the information was collected pertaining to the evidence item in Epoch-Based Date/Time format as specified in Section 3.4.2 of {{RFC8949}}.
```

Comment (henkbirkholz) [reply draft-ietf-sacm-coswid.md]: lgtm

**Comment by henkbirkholz** [comment]
PR summarized in https://mailarchive.ietf.org/arch/msg/sacm/QLWjAZmBcUk96wmgPnvTsOHJNPM/ for WG feedback.

**Review by henkbirkholz — COMMENTED** [review]
submitted as -22

**Pull request merged**